src/entry.rs

```diff
@@ -300,6 +300,46 @@ impl<'a> EntryFields<'a> {
         self.read_to_end(&mut v).map(|_| v)
     }
 
+    pub fn is_pax_sparse(&mut self) -> bool {
+        if let Some(ref pax) = self.pax_extensions {
+            let mut extensions = PaxExtensions::new(pax).filter_map(|f| f.ok());
+            return extensions
+                .find(|f| f.key_bytes() == b"GNU.sparse.major" && f.value_bytes() == b"1")
+                .is_some()
+                && extensions
+                    .find(|f| f.key_bytes() == b"GNU.sparse.minor" && f.value_bytes() == b"0")
+                    .is_some();
+        }
+        false
+    }
+
+    pub fn pax_sparse_name(&mut self) -> Option<Vec<u8>> {
+        if let Some(ref pax) = self.pax_extensions {
+            return PaxExtensions::new(pax)
+                .filter_map(|f| f.ok())
```

Comment:
I'm not a big fan of "swallowing" errors like this, my preference would be to make this function return `Result<Option<Vec<u8>>>` and propagate this error instead.

Reply:
I don't understand. To propagate the error using `Result` we will be dropping the `Vec`.